.github/workflows/bundler-integrations/vite.config.mjs

```diff
@@ -1,15 +1,22 @@
-/** @type {import('vite').UserConfig} */
-// the default vite config used by the different test case configs
+// Allows to run the same configuration against different files
+// as Vite would split shared code in a separate chunk if trying
+// to build an array of entries in `build.rollupOptions.input`
+// or `build.lib.entry`
+const entryName = process.env.ENTRY_NAME ?? 'single-component'
```

Comment:
**note** Set a default here to avoid things breaking badly when runing Vite outside of `npm run vite -w @govuk-frontend/bundler-integrations`